docs/source/iroha_2_whitepaper.md

```diff
@@ -339,7 +339,7 @@ To retrieve information about World State View of the Peer clients will use Iroh
 
 ### 2.14 Client API
 
-Client libraries interact with Iroha over the HTTP and WebSocket. Check the up to date API reference [here](./references/api_spec.md).
+Client libraries interact with Iroha over the HTTP and WebSocket. Check the up-to-date [API Reference](https://hyperledger.github.io/iroha-2-docs/api/torii-endpoints).
```

Comment:
I would change "up-to-date" for "latest", or omit it altogether.
Or we can aim to use the same structure for all of the similar sentences that direct a reader to other parts of the documentation.

In most cases, it is possible to use the following structure:
`<To learn more/For instructions> about/on <topic/term/action>, see [topic](link)`

Examples from the "Security" section:
- To learn more about generating cryptographic keys with `kagami`, available algorithms, and other parameters, see [Generating Cryptographic Keys with Kagami](../security/generating-cryptographic-keys.md#kagami).
- to learn more about cryptographic keys, see [Public Key Cryptography](./security/public-key-cryptography.md)

---

Here, for example, we can try the following:
"To learn more about the available endpoints and operations, see [API > Torii Endpoints](https://hyperledger.github.io/iroha-2-docs/api/torii-endpoints)

Reply:
Applied